submissions/ses3332002/html-css-popup/index.html

```diff
@@ -3,116 +3,120 @@
-              <img src="build/icons/contacts.png" alt="Contacts">
-              Contacts
-            </div>
-            <div class="icon_card" tabindex="0">
-              <img src="build/icons/tanslation.png" alt="Translate">
-              Translate
-            </div>
-            <div class="icon_card" tabindex="0">
-              <img src="build/icons/videos.png" alt="Videos">
-              Videos
-            </div>
-            <div class="icon_card" tabindex="0">
-              <img src="build/icons/search.png" alt="Search">
-              Search
-            </div>
-            <div class="icon_card" tabindex="0">
-              <img src="build/icons/photos.png" alt="Photo">
-              Photo
-            </div>
-            <div class="icon_card" tabindex="0">
-              <img src="build/icons/apps.png" alt="Apps">
-              Apps
-            </div>
-            <div class="icon_card" tabindex="0">
-              <img src="build/icons/documents.png" alt="Document">
-              Document
-            </div>
-          </div>
-          <div class="first_page__footer">
-            <input type="checkbox" class="more_apps__checkbox" id="m_a_checkbox">
-            <label class="more_apps__label" for="m_a_checkbox">More</label>
-            <div class="second_page">
-              <div class="icon_card" tabindex="0">
-                <img src="build/icons/chat.png" alt="Chat">
-                Chat
-              </div>
-              <div class="icon_card" tabindex="0">
-                <img src="build/icons/classroom.png" alt="Classroom">
-                Classroom
-              </div>
-              <div class="icon_card" tabindex="0">
-                <img src="build/icons/collection.png" alt="Collection">
-                Collection
-              </div>
-              <div class="icon_card" tabindex="0">
-                <img src="build/icons/mars.png" alt="Mars">
-                Mars
-              </div>
-              <div class="icon_card" tabindex="0">
-                <img src="build/icons/blogs.png" alt="Blogs">
-                Blogs
-              </div>
-              <div class="icon_card" tabindex="0">
-                <img src="build/icons/horn.png" alt="Ads">
-                Ads
+    <nav>
+      <ul class="user_menu">
+        <li class="user_menu__element user_menu__element-text" >
+          <a href="#">Mail</a>
+        </li>
+        <li class="user_menu__element user_menu__element-text">
+          <a href="#">Images</a>
+        </li>
+        <li>
+          <input type="checkbox" class="apps_menu__checkbox" id="apps_checkbox">
+          <label for="apps_checkbox" class="user_menu__element user_menu__element-anchor">
+            <img src="build/icons/apps_grid.png" alt="apps">
+          </label>
+          <div class="apps_menu">
+            <div class="apps_menu__content">
+              <div class="first_page">
+                <div class="icon_card" tabindex="0">
```

Comment:
Use link `<a>` instead of `<div>` and you don't need to use `tabindex attribute`.

Reply:
I've replaced div's with a's.